app/styles/component/_site-head.scss

```diff
@@ -147,5 +159,38 @@ body:not(.no-scroll) {
 		.site-head {
 			position: fixed;
 		}
+
+		.site-head__partner-slot {
+			position: fixed;
+		}
 	}
 }
+
+.site-head__partner-slot {
+	background: white;
+	border-bottom: 1px solid $vapor;
+	height: $partner-slot-height;
+	position: fixed;
```

Comment:
can this slot be rendered inside an element that is already fixed?
we we have only one fixed element?